filebeat/scripts/generator/fileset/main.go

```diff
@@ -29,7 +29,7 @@ func copyTemplatesToDest(templatesPath, name, filesetPath, module, fileset strin
 
 func generateModule(module, fileset, modulePath, beatsPath string) error {
 	p := path.Join(modulePath, "module", module)
-	if _, err := os.Stat(p); os.IsExist(err) {
+	if _, err := os.Stat(p); !os.IsNotExist(err) {
 		return fmt.Errorf("module already exists: %s at %s", module, p)
```

Comment:
Could add code to return from here silently?

Reply:
Done.